### PR TITLE
CORE-1181: added a function to allow configuration settings to be loa…

### PR DESCRIPTION
**Pull request by slr71**
…ded from a map instead of a file

The biggest changes here were the updates to the unit tests. The actual code changes are pretty straightforward.

### CODE REVIEW

**Comment by slr71** [comment]
Hmm, should we rename this library from `cyverse-de/iplant-clojure-commons` to simply `cyverse-de/clojure-commons` at this time?

**Comment by slr71** [comment]
I went ahead and renamed it. There wasn't much left to do aside from change the repository name anyway.

**Review by ianmcorvidae — APPROVED** [review]
Looks good to me. I like moving things around a little to clarify what's for tests as you've done too.

**Comment by slr71** [comment]
Thanks for the review!

File: test-resources/config.properties [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#Mon Nov 30 16:44:58 MST 2020
+enabled-vector=z'bang, zoom, boing
```
Comment (psarando) [inline test-resources/config.properties:2]: 😆

**Review by psarando — COMMENTED** [review]
LGTM 👍

**Comment by slr71** [comment]
Thanks for the review!

**Pull request merged**